ansible/roles/packages/tasks/install-redhat.yaml

```diff
@@ -18,13 +14,12 @@
       - "python{{ '3' if ansible_distribution_major_version == '8' }}-requests"
       - socat
       - sysstat
-      - yum-utils
-      - yum-plugin-versionlock
       - nfs-utils
       - NetworkManager
     state: present
     enablerepo: "{{ 'offline' if offline_mode_enabled else '' }}"
     disablerepo: "{{ '*' if offline_mode_enabled else '' }}"
+    update_cache: yes
```

Comment:
was getting intermittent failures of the dependent packages, even if they present in the repository.